src/librustc_mir/borrow_check/diagnostics/mutability_errors.rs

```diff
@@ -448,6 +444,100 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
 
         err.buffer(&mut self.errors_buffer);
     }
+
+    /// Targetted error when encountering an `FnMut` closure where an `Fn` closure was expected.
+    fn expected_fn_found_fn_mut_call(&self, err: &mut DiagnosticBuilder<'_>, sp: Span, act: &str) {
+        err.span_label(sp, format!("cannot {ACT}", ACT = act));
```

Comment:
Any reason for using a named parameter here?

Reply:
That was the code already there ^_^'

Reply:
Let's just use `{}`: the named parameter doesn't aid readability at all here :P